src/components/MapCard/DiscountCard/DiscountFunction.tsx

```diff
@@ -0,0 +1,27 @@
+import restaurantInfoStore from '@stores/restaurentStore'
+import { useStore } from '@stores/mapStore'
```

Comment:
```suggestion
import { mapStore } from '@stores/mapStore'
```
이 부분 이렇게 import 하시고 싶으신게 맞나요??